src/clashing_inputs.rs

```diff
@@ -16,7 +16,7 @@ use std::cmp::Ordering;
 /// How should clashing inputs by handled by an [`InputMap`]?
 ///
 /// Inputs "clash" if and only if one [`UserInput`] is a strict subset of the other.
-/// By example:
+/// For example:
```

Comment:
This is a slightly different meaning: "by example" implies "let me prove this", while "for example" means "let me teach you".